reference/multipart.md

```diff
@@ -52,9 +54,14 @@ class NamedPartsExample < Praxis::Types::MultipartArray
 end
 {% endhighlight %}
 
-Parts can also be defined to contain files (and match a filename). To do that you can pass the `filename: true` option when defining the part (or equivalently use the `file` method instead for syntactic sugar). Use the `filename` DSL inside the part definition to express any expected string or pattern for it.
+Parts can also be defined to contain files using the `filename` option.
+This option can take the value `true` or contain a string or regular expression to be matched
```

Comment:
The option itself may only take `true`, which defaults to declaring the filename is a String with `required: true`. 

It's the `filename` method in the part dsl that lets you define the filename attribute in an identical manner to other attributes.

So passing the `filename: true` option is identical to `filename String, required: true` in the DSL.